index-pl.html

```diff
@@ -149,12 +149,12 @@ <h2>Dlaczego maski chronią nas podwójnie</h2>
   
   <div class="container">
 
-    <p>What if both the contagious <i>and</i> the susceptible person wear a mask?</p> 
+    <p>Co jeśli zarówno osoba zakażona jak <i>i</i> niezakażona noszą maski?</p> 
 
     <p>
-      Well, the first mask cuts the chance of infection in half, and the second mask once again cuts the chance of infection in half.
-      So when <i>both</i> people wear masks, the chance of infection is half of half, i.e. 25% (as compared to when neither wear masks).
-      That's a 75% drop in the chance of infection.
+      Pierwsza maska zmniejsza szansę infekcji o połowę, i druga maska także zmniejsza ryzyko infekcji o połowę.
+      Więc jeśli <i>dwie</i> osoby nosza maski, szansa na infekcję wynosi połowę z połowy, np. 25% (w porównaniu do sytuacji gdy nikt nie nosi maski).
```

Comment:
1) literówka, "nosza"
2) i.e. oznacza "to jest", nie "na przykład". Na temat pisowni tj.: https://sjp.pwn.pl/poradnia/haslo/t-j;1865.html